cmd/xdpcap/flags.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type actionsFlag []xdpAction
+
+func (a *actionsFlag) Set(val string) error {
+	// Clear defaults
+	*a = []xdpAction{}
+
+	for _, action := range strings.Split(val, ",") {
+		xdpAction, err := parseAction(action)
+		if err != nil {
+			return err
+		}
+
+		*a = append(*a, xdpAction)
+	}
+
+	return nil
+}
+
+func (a *actionsFlag) String() string {
+	strs := []string{}
+	for _, action := range *a {
+		strs = append(strs, action.String())
+	}
+
+	return strings.Join(strs, ",")
+}
+
+type flags struct {
+	*flag.FlagSet
+
+	mapPath  string
+	pcapFile *os.File
+
+	quiet bool
+	flush bool
+
+	filterExpr string
+	filterOpts filterOpts
+}
+
+// parseFlags creates the flags, and attempts to parse args.
+// Returned flags will never be nil, even on error.
```

Comment:
Nit: It's not a pointer, so it can't be nil?

Reply:
Oops, yes. Fixed